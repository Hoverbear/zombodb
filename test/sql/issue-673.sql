select count(*) from so_posts where so_posts ==> 'body:(the w/3 JAVA)';
select count(*) from so_posts where so_posts ==> 'body:(the w/3 JAVA*)';
select count(*) from so_posts where so_posts ==> 'body:(the w/3 JA?A)';
select count(*) from so_posts where so_posts ==> 'body:(the w/3 JAVA~22)';