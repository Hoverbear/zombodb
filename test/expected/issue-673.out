select count(*) from so_posts where so_posts ==> 'body:(the w/3 JAVA)';
 count 
-------
  4023
(1 row)

select count(*) from so_posts where so_posts ==> 'body:(the w/3 JAVA*)';
 count 
-------
  6084
(1 row)

select count(*) from so_posts where so_posts ==> 'body:(the w/3 JA?A)';
 count 
-------
  4023
(1 row)

select count(*) from so_posts where so_posts ==> 'body:(the w/3 JAVA~22)';
 count 
-------
  4023
(1 row)

